src/SUFST/src/can_unpack.c

```diff
@@ -97,7 +117,9 @@ void queue_receive_thread_entry(ULONG input)
         {
             return;
         }
-
+        /* For statistic */
+        unpack_ptr->stats.rx_can_count++;
+        unpack_ptr->stats.rxbits += sizeof(*rx_msg_ptr);
```

Comment:
`sizeof(rtcan_msg_t)` is not quite what we want here. `rtcan_msg_t` includes some internal data such as the reference count which is not part of the CAN message and shouldn't contribute to the calculation.

I would just suggest to use the length field of `rtcan_msg_t` which is taken from the DLC of the CAN message. Possibly then we should just ignore the CAN message header size in this calculation, since RTCAN strips that out of the message.